pkg/security/apparmor/validate.go

```diff
@@ -106,11 +105,6 @@ func validateHost(runtime string) error {
 		return errors.New("AppArmor is not enabled on the host")
 	}
 
-	// Check runtime support. Currently only Docker is supported.
-	if runtime != kubetypes.DockerContainerRuntime && runtime != kubetypes.RemoteContainerRuntime {
-		return fmt.Errorf("AppArmor is only enabled for 'docker' and 'remote' runtimes. Found: %q", runtime)
-	}
-
```

Comment:
Maybe we should keep it, check the case where runtime is not remote

Reply:
![image](https://user-images.githubusercontent.com/90498607/148909580-088f68f4-4add-4b2f-8714-aa964f6c55ab.png)
refer to #107299